Modules/Loadable/Markups/Resources/Schema/markups-schema-v1.0.2.json

```diff
@@ -0,0 +1,618 @@
+                                    "maxItems": 3
+                                },
+                                "normal": {
+                                    "$id": "#markup/normal",
+                                    "type": "array",
+                                    "title": "Normal",
+                                    "description": "The normal direction of plane markups.",
+                                    "examples": [[0.0, 0.0, 1.0]],
+                                    "additionalItems": false,
+                                    "items": { "type": "number" },
+                                    "minItems": 3,
+                                    "maxItems": 3
+                                },
+                                "size": {
+                                    "$id": "#markup/size",
+                                    "type": "array",
+                                    "title": "Size",
+                                    "description": "The size of the markups representation. Ex. size of the ROI or plane markups.",
+                                    "examples": [[5.0, 5.0, 4.0], [5.0, 5.0, 0.0]],
+                                    "additionalItems": false,
+                                    "items": { "type": "number" },
+                                    "minItems": 3,
+                                    "maxItems": 3
+                                },
+                                "objectToBase": {
+                                    "$id": "#markup/objectToBase",
+                                    "type": "array",
+                                    "title": "Object to Base matrix",
+                                    "description": "4x4 transform matrix from the object representation to the coordinate system defined by the control points.",
+                                    "examples": [[-0.9744254538021788, -0.15660098593235834, -0.16115572030626558, 26.459385388492746,
+                                                  -0.08525118065879463, -0.4059244688892957, 0.9099217338613386, -48.04154530201596,
+                                                  -0.20791169081775938, 0.9003896138683279, 0.3821927158637956, -53.35829266424462,
+                                                  0.0, 0.0, 0.0, 1.0]],
+                                    "additionalItems": false,
+                                    "items": { "type": "number" },
+                                    "minItems": 16,
+                                    "maxItems": 16
+                                },
+                                "orientation": {
+                                    "$id": "#markup/orientation",
+                                    "type": "array",
+                                    "title": "Markups orientation",
+                                    "description": "3x3 orientation matrix of the markups representation in node coordinates.",
```

Comment:
to clarify this, add something like "(orientation[0], orientation[3], orientation[6]) is the x vector of the ... coordinate system in the ... coordinate system"